rad/prelude/strings.rad

```diff
@@ -107,6 +108,28 @@
   [ (lines " \n \t") ==> [" " " \t"]]
   [ (lines "") ==> []])
 
+;; pad-to
+
+(def pad-to
+  "Appends the `word` with whitespace to get to length `l`. If `word` is longer
```

Comment:
Don't most "padding" function just return the input rather than truncate?

Reply:
You are right. We need the truncate-or-pad functionality to format the list tables. Would `pad-or-truncate-to` work, or can you think of a shorter function name?

Reply:
I would probably implement `pad` and then just use `(take _ (pad _))` at the call site. If it is used many times maybe call it `pad-truncate`.

Reply:
I like the `(take _ (pad _))` option. It is used only twice so far. A more accurate naming would be `pad-right-to` probably.